ios/ApiMagnetReact.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ApiMagnetReact.swift
+//  Magnet
+//
+//  Created by Francisco Jordano on 01/11/2016.
+//  Copyright © 2016 Mozilla. All rights reserved.
+//
+
+import Foundation
+
+@objc(ApiMagnetReact) class ApiMagnetReact: NSObject {
+  private let api: ApiMagnet = ApiMagnet()
+  
+  @objc func get(path: String,
+                 resolve: RCTPromiseResolveBlock,
+                 reject: RCTPromiseRejectBlock) {
```

Comment:
Sure, with the longer names in the previous signature was a pain, but now we can go back to normal formatting